### PR TITLE
Improves validate-cocina-roundtrip diffing.

### LINKED ISSUE #1471: Improve validate-cocina-roundtrip diffing
This is to make it easier for developer to evaluate differences when roundtripping.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #1471

## Why was this change made?
To make it easier to understand the differences when roundtripping.

## How was this change tested?
Locally

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

**Comment by ndushay** [comment]
I went to sdr-deploy.  After this change, an individual druid result file ends thus:

SO MUCH BETTER

```
Differences 11
Original node:
<recordInfo>
        <languageOfCataloging usage="primary">
          <languageTerm type="text" authority="iso639-2b" authorityURI="http://id.loc.gov/vocabulary/iso639-2" valueURI="http://id.loc.gov/vocabulary/iso639-2/eng">English</languageTerm>
          <languageTerm type="code" authority="iso639-2b" authorityURI="http://id.loc.gov/vocabulary/iso639-2" valueURI="http://id.loc.gov/vocabulary/iso639-2/eng">eng</languageTerm>
        </languageOfCataloging>
        <recordContentSource authority="marcorg" authorityURI="http://id.loc.gov/vocabulary/organizations" valueURI="http://id.loc.gov/vocabulary/organizations/cst">CSt</recordContentSource>
        <recordOrigin>human prepared</recordOrigin>
      </recordInfo>
Best guess roundtripped node:
<recordInfo>
    <languageOfCataloging usage="primary">
      <languageTerm valueURI="http://id.loc.gov/vocabulary/iso639-2/eng" authorityURI="http://id.loc.gov/vocabulary/iso639-2" authority="iso639-2b" type="text">English</languageTerm>
      <languageTerm valueURI="http://id.loc.gov/vocabulary/iso639-2/eng" authorityURI="http://id.loc.gov/vocabulary/iso639-2" authority="iso639-2b" type="code">eng</languageTerm>
    </languageOfCataloging>
    <recordContentSource valueURI="http://id.loc.gov/vocabulary/organizations/cst" authorityURI="http://id.loc.gov/vocabulary/organizations" authority="marcorg">CSt</recordContentSource>
    <recordOrigin>human prepared</recordOrigin>
  </recordInfo>
Difference:
 <recordInfo>
-        <languageOfCataloging usage="primary">
-          <languageTerm type="text" authority="iso639-2b" authorityURI="http://id.loc.gov/vocabulary/iso639-2" valueURI="http://id.loc.gov/vocabulary/iso639-2/eng">English</languageTerm>
-          <languageTerm type="code" authority="iso639-2b" authorityURI="http://id.loc.gov/vocabulary/iso639-2" valueURI="http://id.loc.gov/vocabulary/iso639-2/eng">eng</languageTerm>
-        </languageOfCataloging>
-        <recordContentSource authority="marcorg" authorityURI="http://id.loc.gov/vocabulary/organizations" valueURI="http://id.loc.gov/vocabulary/organizations/cst">CSt</recordContentSource>
-        <recordOrigin>human prepared</recordOrigin>
-      </recordInfo>
\ No newline at end of file
+    <languageOfCataloging usage="primary">
+      <languageTerm valueURI="http://id.loc.gov/vocabulary/iso639-2/eng" authorityURI="http://id.loc.gov/vocabulary/iso639-2" authority="iso639-2b" type="text">English</languageTerm>
+      <languageTerm valueURI="http://id.loc.gov/vocabulary/iso639-2/eng" authorityURI="http://id.loc.gov/vocabulary/iso639-2" authority="iso639-2b" type="code">eng</languageTerm>
+    </languageOfCataloging>
+    <recordContentSource valueURI="http://id.loc.gov/vocabulary/organizations/cst" authorityURI="http://id.loc.gov/vocabulary/organizations" authority="marcorg">CSt</recordContentSource>
+    <recordOrigin>human prepared</recordOrigin>
+  </recordInfo>
\ No newline at end of file
```

File: bin/validate-cocina-roundtrip [hunk 1]
```diff
@@ -24,13 +26,28 @@ cache = FedoraCache.new
...
+  new_ng_xml = normalize_topics(new_ng_xml)
+  new_ng_xml = normalize_subject_name(new_ng_xml)
+  new_ng_xml = normalize_authority_uris(new_ng_xml)
+  new_ng_xml = normalize_origin_info_event_types(new_ng_xml)
+  normalize_subject_authority(new_ng_xml)
+end
+
+def normalize_blanks(ng_xml)
+  Nokogiri::XML(ng_xml.to_s) { |config| config.default_xml.noblanks }
+end
```
Comment (mjgiarlo) [inline bin/validate-cocina-roundtrip:41]: What calls this method?

**Review by mjgiarlo — APPROVED** [review]
Thanks for working on this!

File: bin/validate-cocina-roundtrip (hunk 1 above)
Comment (justinlittman) [reply bin/validate-cocina-roundtrip]: Nada. Removed.

**Pull request merged**